packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableSelectableDemo.tsx

```diff
@@ -47,7 +45,7 @@ export class TableSelectableDemo extends React.Component<TableProps, TableState>
     this.onSelect = this.onSelect.bind(this);
   }
 
-  onSelect(event: React.MouseEvent, isSelected: boolean, rowId: number) {
+  onSelect(event: React.FormEvent<HTMLInputElement>, isSelected: boolean, rowId: number) {
```

Comment:
We use `onSelect` in Cost Management. Technically, this would be a breaking change for the build.

If you look at the event in the debugger, it is `MouseEvent`.

Reply:
Yes I see what you mean, it was generally the purpose of this PR to update the type to FormEvent so that we get correct type checking when attempting to use the `checked` property from the event target. Seems like I'm missing something?

I'm not really sure what's being suggested, can you help me understand? Happy to do whatever is needed to fix the problem.

Reply:
When I look at the `onSelect` function, the `event` being passed in is of type `MouseEvent` (i.e., when I select any checkbox). In Cost Management, our handleOnSelect callback also defines this as `event: MouseEvent`. I'm simply suggesting that using `FormEvent` would be a breaking change.

Reply:
Thanks for clarifying!